packages/slice-machine/lib/mock/Slice.ts

```diff
@@ -61,19 +60,11 @@ export function buildSliceMockConfig(
 export default function MockSlice(
   smModel: SliceSM,
   legacyMockConfig: Record<string, Record<string, Record<string, unknown>>> // not sure about this one.
-): Array<SharedSliceContent> {
+): ComponentMocks {
   const model = Slices.fromSM(smModel);
 
   const sliceMockConfig = buildSliceMockConfig(model, legacyMockConfig);
-  const sliceModel: SharedSlice = {
-    ...model,
-    variations: model.variations.map((v) => {
-      return {
-        ...v,
-        imageUrl: "",
```

Comment:
why did we strip out the imageUrl before?

Reply:
We use to require it, now we have a fallback system so if not specified, the validator will automatically set a default url that is used in the wroom already